### DISCUSSION TITLE
CodeMirror Syntax Highlighting Script Blocked

### DISCUSSION THREAD

**Discussion by RosanneUssery** [community/community#40318] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

For my team’s use case, we wanted to create an easy “fix the issue we are identifying for you” button on one of our in-house PR checks. To do so, we link to the appropriate file/repo for them to edit it directly on GitHub (like so: `https://github.com/${owner}/${repo}/edit/${branch}/${path_to_file_with_type}`). 

This works great in general, the only issue is that on the redirect the syntax highlighting does not load. The only way to make it available appears to be to refresh the page. Looking a little closer, it appears that when the file is linked to in the above manner, it runs up against [this script-src CSP error](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/script-src):

```markdown
loadmode.ts:85 Refused to load the script 
'https://github.com/static/javascripts/codemirror/mode/yaml/yaml.js?v=1' 
because it violates the following Content Security Policy directive: 
"script-src github.githubassets.com". 
Note that 'script-src-elem' was not explicitly set, so 'script-src' is used as a fallback. 
```

If possible, it would be great to have these scripts added to the approved source so we have immediate syntax highlighting.

Thanks!

**Comment by ossanna16** [comment]
Hi there @RosanneUssery and welcome to our community! Thank you for reporting this!

**Reply by RosanneUssery** [reply]
Thank you very much for your warm welcome! Glad to help.
